rm_vk/rm_vk/NetworkService/RequestType.swift

```diff
@@ -0,0 +1,24 @@
+// RequestType.swift
+// Copyright © RoadMap. All rights reserved.
+
+/// типы запросов
+enum RequestType {
+    case friends
+    case groups
+    case photos(id: Int)
+    case searchGroups(searchQuery: String)
+
+    var urlString: String {
+        switch self {
+        case .friends:
+            return Api.baseUrl + "friends.get?" + Api.acessToken + Api.friendsFields + Api.version
```

Comment:
```suggestion
            return Api.baseUrl + "" + Api.acessToken + Api.friendsFields + Api.version
```
убираем текстовки, конкатенацию меняем на интерполяцию